extension/src/main/scala/io/scalac/mesmer/extension/upstream/LabelsFactory.scala

```diff
@@ -1,48 +1,71 @@
 package io.scalac.mesmer.extension.upstream
 
-import io.opentelemetry.api.metrics.common.Labels
+import io.opentelemetry.api.common.AttributeKey
+import io.opentelemetry.api.common.Attributes
 
 import scala.annotation.switch
-import scala.collection.mutable.ArrayBuffer
 
 object LabelsFactory {
```

Comment:
The name should change to `AttributesFactory` imho. It seems like `Labels` is a deprecated concept in opentelemetry specification now (I have to make sure)

Reply:
this commit solves it: 202648586b775bae3ef9ee6f6cd61bf20acb5116